tests/conftest.py

```diff
@@ -21,3 +23,12 @@ def event_loop() -> Iterator[TimeForwardLoop]:
     loop = TimeForwardLoop()
     yield loop
     loop.close()
+
+
+@pytest.fixture
+def frozen_time() -> Iterator[freezegun.api._freeze_time]:
```

Comment:
how about reimporting somewhere with `from freezegun.api import _freeze_time as FreezeTime` so this is nice to handle with in the tests?